### PR TITLE
Responsive grid using tailwind

### PR DESCRIPTION
**Pull request by AdamDotCom**
"Tailwind CSS works by scanning all of your HTML files, JavaScript components, and any other templates for class names, generating the corresponding styles and then writing them to a static CSS file.

It's fast, flexible, and reliable — with zero-runtime." https://tailwindcss.com/docs/installation

Related PR: https://github.com/zer0-os/zOS-theme-engine/pull/6

### CODE REVIEW

**Review by jtulloch — CHANGES_REQUESTED** [review]
so, general thought is that this seems fine, but we should keep it isolated to layout. since markup and layout styles always directly correspond, i think it makes sense to have those things together.

File: src/index.tsx [hunk 1]
```diff
@@ -36,9 +36,18 @@ ReactDOM.render(
 );
 
 ReactDOM.render((
+  <div className='flex flex-row'>
     <Router history={history}>
-      <AppSandboxContainer store={store} />
+      <div className='basis-[3.5rem]'>{/* World navigation */}</div>
```
[the remark this answers is not in the document]
Comment (AdamDotCom) [reply src/index.tsx:41]: It is possible to assign a class of say `world-navigation` to `basis-[xyz]` if numbers are visual clutter. CSS variables can also be used to set the numeric values if we want the values to be defined somewhere else altogether.
 
Tailwind has media query capabilities through class prefixes defined similar to `md:world-navigation`, this allows expand / collapse, buttons customization etc... if you wanted to use Tailwind outside of layout markup.

File: src/Main.tsx [hunk 2]
```diff
@@ -13,12 +13,16 @@ import './main.scss';
...
   render() {
     return (
-      <div className='main'>
-        <div className='main__header'>
-          <Link className='main__title' to={`/${config.defaultZnsRoute}`}>
-            <span>zOS</span>
+      <div className='flex flex-row items-center h-[64px] px-[3.5rem] space-x-1 main__header'>
+        <div className='justify-center basis-[20px]'>{/* World image */}
+          <Link to={`/${config.defaultZnsRoute}`}>
+            <span className='text-3xl'>zOS</span>
```
[the remark this answers is not in the document]
Comment (AdamDotCom) [reply src/Main.tsx:19]: “i don’t think we want to give up the benefits of the C part of CSS” Sounds like you need to get yourself on a pulpit! Preach it! :)

`text-3xl` shouldn't have been there. It was a fragment from getting tailwind configured.

[the remark this answers is not in the document]
Comment (AdamDotCom) [reply src/Main.tsx:19]: I'm not sure where we'd lose the Cascade though, font could be cascading from the parent. If you wanted to apply a unique font the Tailwind way you'd likely have `text-3xl font-sans`. 

`text-3xl` corresponds to:
```
.text-3xl {
  font-size: 1.875rem;
  line-height: 2.25rem;
}
```

**Pull request closed without merging**